data/json/items/book/tailor.json

```diff
@@ -35,6 +35,14 @@
     "fun": 2
   },
   {
+    "id": "mag_animecon",
+    "copy-from": "mag_tailor",
+    "type": "BOOK",
+    "name": "Boston AnimeCon magazine",
+    "name_plural": "Boston AnimeCon magazines",
```

Comment:
This is useless entry since the general rule to define plural name (name+"s") gives the same result.